ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -1,8 +1,153 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
 import ZoneId from '../../../../../resources/zone_id.js';
 
 export default {
   zoneId: ZoneId.EdensPromiseUmbra,
   timelineFile: 'e9n.txt',
   triggers: [
+    {
+      id: 'E9N Ground-Razing Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '55ED', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N The Art Of Darkness Right',
+      netRegex: NetRegexes.startsUsing({ id: '5223', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 4,
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'E9N The Art Of Darkness Left',
+      netRegex: NetRegexes.startsUsing({ id: '5224', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 4,
+      response: Responses.goRight(),
+    },
+    {
+      id: 'E9N Wide-Angle Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '5AFF', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.awayFromFront('info'),
+    },
+    {
+      id: 'E9N Zero-Form Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '55EB', source: 'Cloud Of Darkness' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E9N Empty Plane',
+      netRegex: NetRegexes.startsUsing({ id: '4FC6', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9N Mire Of Despair',
+      netRegex: NetRegexes.startsUsing({ id: '5B07', source: 'Cloud Of Darkness', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Out from Under',
```

Comment:
Sorry, maybe I'm tired and don't remember what this mechanic is, but what does this mean? Is this getOut? or outOfMelee? or?

Reply:
If I'm remembering it correctly, this is during the tile phase when the center tile is gone? I'm not sure either of those responses would be correct enough in this situation.

Reply:
During the tile phase, the center tile becomes an insta-death tile, yea. I tried to reuse something, but I don't know if it makes enough sense?

Reply:
Looking at the log and video, I think there's a visible ground aoe here that says don't stand in it, so maybe this trigger isn't needed?